activejob/lib/active_job/railtie.rb

```diff
@@ -43,7 +43,7 @@ class Railtie < Rails::Railtie # :nodoc:
       end
 
       ActiveSupport.on_load(:active_record) do
-        self.destroy_association_async_job = ActiveRecord::DestroyAssociationAsyncJob
+        self.destroy_association_async_job ||= ActiveRecord::DestroyAssociationAsyncJob
```

Comment:
I noticed that the similar [`config.action_mailer.delivery_job`](https://guides.rubyonrails.org/configuring.html#config-action-mailer-delivery-job) configuration calls `constantize` on the passed in value.

https://github.com/rails/rails/blob/67bffae6c7c35a34adc5eecada52dee04f2ad21c/actionmailer/lib/action_mailer/railtie.rb#L45-L47

That makes it seem like it's expecting a string rather than a class. Should this configuration also expect a string (or accept a string or a class)? I noticed when I tried to use this setting in `config/application.rb`, I couldn't use job classes from `app/jobs` because they hadn't been loaded yet. 🤔